src/index.ts

```diff
@@ -29,50 +29,48 @@ export type Account = {
   debits_accepted: bigint // u64
   credits_reserved: bigint // u64
   credits_accepted: bigint // u64
+  timestamp: bigint // u64, Set this to 0n - the actual value will be set by TigerBeetle server
 }
 
-export type CreateAccount = Account & {
-  timestamp: bigint // u64
-}
-
-export enum CreateAccountFlags {
+export enum AccountFlags {
   linked = (1 << 0),
   debits_must_not_exceed_credits = (1 << 1),
   credits_must_not_exceed_debits = (1 << 2)
 }
 
 export enum CreateAccountError {
-    linked_event_failed = 1,
-    exists,
-    exists_with_different_user_data,
-    exists_with_different_reserved_field,
-    exists_with_different_unit,
-    exists_with_different_code,
-    exists_with_different_flags,
-    exceeds_credits,
-    exceeds_debits,
-    reserved_field,
-    reserved_flag_padding,
+  linked_event_failed = 1,
+  exists,
+  exists_with_different_user_data,
+  exists_with_different_reserved_field,
+  exists_with_different_unit,
+  exists_with_different_code,
+  exists_with_different_flags,
+  exceeds_credits,
+  exceeds_debits,
+  reserved_field,
+  reserved_flag_padding,
 }
 
-export type CreateAccountResult = {
+export type CreateAccountsResult = {
```

Comment:
Let's do `CreateAccountsError` here seeing as we're moving to errors over results.